### PR TITLE
Release prep 2022-06-07

### PR DESCRIPTION
**Pull request by xenoworf**
Release prep.

### CODE REVIEW

File: packages/terra-dev-site/package.json [hunk 1]
```diff
@@ -94,4 +94,4 @@
   "devDependencies": {
     "terra-application": "^2.0.0"
   }
-}
+}
```
Comment (sdadn) [inline packages/terra-dev-site/package.json:97]: nit: newline

File: packages/terra-dev-site/CHANGELOG.md [hunk 2]
```diff
@@ -2,8 +2,11 @@
 
 ## Unreleased
 
+## 8.1.0 - (June 7, 2022)
+
 * Changed
-  * updated `terra-application` version in peer dependency.
```
Comment (BenBoersma) [inline packages/terra-dev-site/CHANGELOG.md:6]: Is the removal of this line intentional?

Comment (xenoworf) [reply packages/terra-dev-site/CHANGELOG.md:6]: Yes. The original line didn't express the new feature (IE 10 support) which justified the minor version bump. So, once I determined this new feature would be pulled in I edited the line to make it more descriptive to our consumers.

If, on the other hand, I'm wrong about IE 10 date-time functions being _used_ in terra-dev-site then I can revert this and instead bump this package as a patch (8.0.1).

**Review by sdadn — APPROVED** [review]

**Review by BenBoersma — APPROVED** [review]

File: packages/terra-dev-site/CHANGELOG.md [hunk 3]
```diff
@@ -2,8 +2,11 @@
...
+  * Support for date-time functions in ie 10.
```
Comment (supreethmr) [inline packages/terra-dev-site/CHANGELOG.md]: terra-dev-site has only peer dependency update changes. : https://github.com/cerner/terra-application/pull/296 . 

date-time support changes are done only for terra-polyfill : https://github.com/cerner/terra-application/pull/297 

So the previous change-log is valid for terra-dev-site : `updated `terra-application` version in peer dependency.`

Comment (xenoworf) [reply packages/terra-dev-site/CHANGELOG.md]: Now I understand, thanks. I'll revert it back to how it see.

**Comment by benbcai** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Please include a CHANGELOG entry for each changed package on this PR. Looks like a CHANGELOG is missing for: 

 - terra-application-docs</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 29db88ce6a083ca4ff42637173af3a91151b18c6
</p>

**Review by supreethmr — APPROVED** [review]

**Pull request merged**